fv3gfs-util/fv3gfs/util/partitioner.py

```diff
@@ -141,17 +143,38 @@ def global_extent(self, rank_metadata: QuantityMetadata) -> Tuple[int, ...]:
             rank_metadata.dims, rank_metadata.extent, self.layout
         )
 
-    def subtile_extent(self, global_metadata: QuantityMetadata) -> Tuple[int, ...]:
+    def subtile_extent(self, global_metadata: QuantityMetadata, rank: int, overlap: bool, cube_metadata_offset=0) -> Tuple[int, ...]:
```

Comment:
What does "cube_metadata_offset" mean? Can you add a list of `Args:` to the docstring where you describe it?

Reply:
I don't think `overlap` should be an argument for this, right? It would always be True, since we want the size of the numpy array/storage on this rank.

Reply:
`cube_metadata_offset` was unhelpfully named. It's now a boolean `is_cube_metadata` that tells `subtile_extent` that the metadata that's passed in is that of a cubed Sphere model. It's used to offset the `dims` and `extent` members of `global_metadata`, like in this call: `global_metadata.dims[cube_metadata_offset:]`.
A docstring has been added.

Reply:
You're right, `overlap` should always be true in that function. Replaced by `True`.